AfterClass-io/afterclass.io-v2
# feat(ui): tooltip component

closes #166 

## Changes
- add tooltip component at `@/common/components/tooltip`
- add tooltip on course code in review item headers. tooltip shows course name

## Implementation Details

- use radix-ui tooltip primitive component, with shadcn's default props
- reduce default tooltip reveal delay to 200ms instead of 700ms

## How to Test


1. head to home page
2. hover over course code of a review
3. see course name appear

## Preview / Screenshots

![image](https://github.com/user-attachments/assets/1adf6bbe-3d72-42cb-af9f-72eaafb1ba35)


## Checklist
